content/applications/sales/point_of_sale/payment/adyen.rst

```diff
@@ -55,6 +55,22 @@ To find this number, go to your **Adyen account**. Then, go to :menuselection:`P
 --> Terminals`, select the terminal to link, and copy-paste its serial number onto the Odoo
 mandatory field.
 
+Set the Event URLs
+~~~~~~~~~~~~~~~~~~
+In order for Odoo to know when a payment have been made, the terminal `Event URLs` needs to be set.
+
+To do this, please follow this steps:
+
+ 1. Connect to your **Adyen account**
```

Comment:
Numbered lists should be written using the #. instead of actual numbers (in case the list has to change afterward)

```suggestion
#. log in on `Adyen's website <https://www.adyen.com/>`_;
```